features/src/main/feature/camel-features.xml

```diff
@@ -97,6 +97,21 @@
         <bundle dependency="true">mvn:jakarta.servlet/jakarta.servlet-api/5.0.0</bundle>
     </feature>
 
+    <feature name="jetty12" version="${jetty-version}">
```

Comment:
Let's keep `jetty` as name as we did for the 2 versions of the feature of `http-client`

Reply:
done, I had tried by fixing the version, it did not work. With version range, it's working